lib/modules/manager/bundler/extract.ts

```diff
@@ -103,11 +104,42 @@ export async function extractPackageFile(
       });
     }
 
+    const gemGitRefsMatchRegex = regEx(
+      `^\\s*gem\\s+(['"])(?<depName>[^'"]+)['"]((\\s*,\\s*git:\\s*['"](?<gitUrl>[^'"]+)['"])|(\\s*,\\s*github:\\s*['"](?<repoName>[^'"]+)['"]))(\\s*,\\s*branch:\\s*['"](?<branchName>[^'"]+)['"])?(\\s*,\\s*ref:\\s*['"](?<refName>[^'"]+)['"])?(\\s*,\\s*tag:\\s*['"](?<tagName>[^'"]+)['"])?`,
+    );
+    const gemGitRefsMatch = gemGitRefsMatchRegex.exec(line);
+
     const gemMatchRegex = regEx(
       `^\\s*gem\\s+(['"])(?<depName>[^'"]+)(['"])(\\s*,\\s*(?<currentValue>(['"])[^'"]+['"](\\s*,\\s*['"][^'"]+['"])?))?`,
     );
     const gemMatch = gemMatchRegex.exec(line);
-    if (gemMatch) {
+
+    if (gemGitRefsMatch) {
+      const dep: PackageDependency = {
+        depName: gemGitRefsMatch.groups?.depName,
+        managerData: { lineNumber },
+      };
+      if (gemGitRefsMatch.groups?.gitUrl) {
```

Comment:
```suggestion
      if (gemGitRefsMatch.gitUrl) {
```
please update following code accordingly 🤗